pages/doc/alerts_states_lifecycle.md

```diff
@@ -202,7 +204,7 @@ By default, the **Resolve Window** is the same length as the **Trigger Window**.
 
 Suppose you define an alert with the following properties:
 * The alert condition is `ts(metric.name) > 0`, where `metric.name` reports once a minute. (The summarization values are therefore the same as the reported values.)
-* The **Checking Frequency** interval is 1 minute (the default).
+* The **Checking Frequency** interval is 1 minute (the default value is 5 minutes).
```

Comment:
Maybe this should say, "The **Checking Frequency** interval is 5 minutes ( the default )..."

Reply:
Hi Jeff,

Thank you so much!
As we discussed. Will leave this change for now because the examples are structured to show 1 second.